### DISCUSSION TITLE
Can you invalidate mulitple tags?

### DISCUSSION THREAD

**Discussion by KaJonesIMI** [reduxjs/redux-toolkit#4113] [Q&A] [answered]
When I tried to do this myself, it didn't work. I was wondering if it's even possible. I was trying
`
invalidatesTags: ["Foo", "Bar"],
`
but it only ever seemed to invalidate Foo.

**Comment by EskiMojo14** [comment, marked as answer]
What you have there should work - do you have a reproduction of it not working?

**Reply by KaJonesIMI** [reply]
Ironically, by confirming my invalidatedTags is correct, it made me look at my tag provider again... which was wrong. So after fixing that syntax error, it's working! Thanks for confirming that I can invalidate multiple tags though :)
